app/js/components/LabOrdersList/LabOrdersList.jsx

```diff
@@ -7,13 +7,131 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 import React from 'react';
+import { connect } from 'react-redux';
+import cn from 'classnames';
+import moment from 'moment';
+import Loading from '../Loading';
+import SortableTable from '../SortableTable';
+import { fetchLabOrders } from '../../actions/labOrderAction';
+import "react-table/react-table.css";
+import "./index.scss";
+
+
+
+const Cell = ({columnName, value}) => {
+  if (columnName === "EMR ID") {
```

Comment:
A Switch statement would be perfect here so we do not have to keep on running the other If blocks even after a match has been found.

Reply:
@jomadoye there's a return statement in each condition block that makes it not to run others, but I agree with your reasoning, so I'll refactor